code/modules/shuttle/computer.dm

```diff
@@ -51,7 +51,7 @@
 	if(href_list["move"])
 		var/obj/docking_port/mobile/M = SSshuttle.getShuttle(shuttleId)
 		if(M.launch_status == ENDGAME_LAUNCHED)
-			to_chat(usr, "<span class='warning'>You've already escaped. Never going back to that place again!</span>")

```

Comment:
this is a joke about one of the ending sounds

Reply:
the ending sound is "not goin back to that place again"